src/bo4e/enum/gebiettyp.py

```diff
@@ -1,9 +1,9 @@
 """
-List of possible Gebietstypen.
+List of possible Gebiettypen.
```

Comment:
Removed **s** in Gebietstypen, cause the documentation calls this enum just Gebiettyp